pyproject.toml

```diff
@@ -28,3 +28,6 @@ dependencies = {file = ["requirements.txt"]}
 
 [tool.black]
 line-length = 79
+
+[tool.mypy]
+ignore_missing_imports = true
```

Comment:
This is a specific mypy flag, that says it shouldn't fail due to imports you use that don't have valid type hints. Since we use such imports (e.g., mona-sdk) - we have to ignore that.